api/src/main/java/com/datastrato/gravitino/SupportsMetalakes.java

```diff
@@ -75,7 +75,7 @@ Metalake alterMetalake(NameIdentifier ident, MetalakeChange... changes)
    * Drop a metalake with specified identifier.
    *
    * @param ident The identifier of the metalake.
-   * @return True if the metalake was dropped, false otherwise.
+   * @return True if the metalake was dropped, false if the metalake does not exist.
    */
   boolean dropMetalake(NameIdentifier ident);
```

Comment:
I see you don't change the metalake and catalog related implementations, can you confirm do they have the correct implementation?